### ISSUE TITLE
[SCREENREADER]: Clock icons MUST have text that describes the info represented by them

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#6697]
# [508-defect-2 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

```diff
-  ** Must** for if the feedback must be applied 
!  **Should** if the feedback is best practice 
+  **Consider** for suggestions/enhancements 
```
## Description

Relevant pages: 
- https://www.va.gov/decision-reviews/multiple-party-claims/
- https://www.va.gov/decision-reviews/

For the grey boxed text beneath step 1 - "Choose one of the 3 options to appeal with a Veterans Law Judge" - for a screen reader the "About 1 year" and "Longer" headings lack the context provided to visual users who are able to see the clock icon. Images **MUST** have text alternatives that describe the information or function represented by them. It was confusing and unclear to hear "About one year" and "Longer" without context. Without my sight, I wouldn't know what it referenced and miss important information relevant to my appeal/decision review.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screenreader user, I want to understand the context of content I hear in order to comprehend what I am reading.

## Environment

* Operating System: any
* Browser: any
* Screenreading device: any
* Server destination: production

## Steps to Recreate

1. Enter `https://www.va.gov/decision-reviews/multiple-party-claims/` in browser
2. Start screenreading device 
3. Navigate to the section "Choose one of the 3 options to appeal with a Veterans Law Judge"
4. Verify that there is no context read to pair with "About one year" or "Longer"

## Possible Fixes (optional)

As these are static pages, it may be most prudent to copyedit the headings so they include context that is visibly implied by the clock icon. For example, the heading could read: How long before the Veterans Law Judge provides a decision? Then, provide the answer.

## WCAG or Vendor Guidance (optional)

* [WCAG Images must have text alternatives that describe the information or function represented by them. ](https://www.w3.org/WAI/tutorials/images/)

## Screenshots or Trace Logs

![Screen Shot 2020-03-09 at 2 24 01 PM](https://user-images.githubusercontent.com/57469/76245097-aa56c200-6211-11ea-916b-0b9030af1931.png)

**Comment by RLHecht** [comment]
@lwalsh617 @MickinSahni These are being fixed once the pages are migrated into Drupal and live on the site. Should we move this out of this sprint until that happens?

**Comment by MickinSahni** [comment]
2020-03-20 PW Standup
* Wasn't in sprint but was in Current Sprint column. Moved to Ready until pages are migrated to Drupal

**Comment by MickinSahni** [comment]
@jenstrickland this shouldn't be an issue any more as the icons have been removed in addition to a sentence for timeframe context.

**Comment by jenstrickland** [comment]
@MickinSahni Is there a URL that I may validate?

**Comment by MickinSahni** [comment]
> @MickinSahni Is there a URL that I may validate?

Yes! The Drupal preview links will no longer show the clock/timeframe, but @lwalsh617 will share the specific links here. 
@jenstrickland

**Comment by lwalsh617** [comment]
Hey @jenstrickland,
Wherever we dropped the clock icon, we added an H2/question above the timing, for example:
**How long does it take VA to make a decision?**
4-5 months
VA's goal for completing Supplemental Claims is an average of 125 days. 

Here are the Drupal preview links for the pages this impacts: 
- [ ] [Decision reviews and appeals](https://preview.prod.vfs.va.gov/preview?nodeId=3071)
- [ ] [Supplemental Claims](https://preview-prod.vfs.va.gov/preview?nodeId=3011)
- [ ] [Higher-Level Reviews](https://preview-prod.vfs.va.gov/preview?nodeId=3003)
- [ ] [Board Appeals](https://preview-prod.vfs.va.gov/preview?nodeId=3029)
- [ ] [After a Board Appeal](https://preview-prod.vfs.va.gov/preview?nodeId=3013)
- [ ] [Contested Claims](https://preview-prod.vfs.va.gov/preview?nodeId=3030)
- [ ] [Fiduciary claims](https://preview-prod.vfs.va.gov/preview?nodeId=3016)
- [ ] [Insurance claims](https://preview-prod.vfs.va.gov/preview?nodeId=3025)

**Comment by jenstrickland** [comment]
I just tested the Insurance claims page for a quick spot check, and love the clarity this provides for those with cognition issues as well as for screen readers. The headings also give those who navigate by headings more insight. Thanks! I will conduct a more thorough review at the appropriate review point, okay, @MickinSahni ?

(Providing the list below for ease of use. Since these have to be viewed in the Windows environment, it isn't possible to click links from the ticket to open. Everything gets manually typed. Fortunately, I can just change the number at the end to speed things up.)

1.  Decision reviews and appeals — https://preview.prod.vfs.va.gov/preview?nodeId=3071
2.  Supplemental Claims — https://preview-prod.vfs.va.gov/preview?nodeId=3011
3.  Higher-Level Reviews — https://preview-prod.vfs.va.gov/preview?nodeId=3003
4.  Board Appeals — https://preview-prod.vfs.va.gov/preview?nodeId=3029
5.  After a Board Appeal — https://preview-prod.vfs.va.gov/preview?nodeId=3013
6.  Contested Claims — https://preview-prod.vfs.va.gov/preview?nodeId=3030
7.  Fiduciary claims — https://preview-prod.vfs.va.gov/preview?nodeId=3016
8.  Insurance claims — https://preview-prod.vfs.va.gov/preview?nodeId=3025

**Comment by MickinSahni** [comment]
Thanks! Closing this out. Have staging review at a later date!
